src/ifElse.test.js

```diff
@@ -1,11 +1,53 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it(`should be declared`, () => {
+    expect(ifElse).toBeInstanceOf(Function);
+  });
+
+  it('should run a first callback if condition returns true', () => {
+    const condition = jest.fn(() => true);
+    const first = jest.fn();
+    const second = jest.fn();
+
+    ifElse(condition, first, second);
+
+    expect(first)
+      .toHaveBeenCalled();
+  });
+
+  it('should not run a second callback if condition returns true', () => {
+    const condition = jest.fn(() => true);
+    const first = jest.fn();
+    const second = jest.fn();
+
+    ifElse(condition, first, second);
 
+    expect(second)
+      .not.toHaveBeenCalled();
   });
 
-  // write tests here
+  it('should run a second callback if condition returns false', () => {
```

Comment:
Use a template literal for consistency with other test descriptions, e.g., `should run the second callback if the condition returns false`